### DISCUSSION TITLE
Best practice for multiple blueprints in a collection

### DISCUSSION THREAD

**Discussion by adamhopkinson** [statamic/cms#8695] [Q&A] [answered]
Say I want to create a blog with several types of post - links, images, videos, etc. Like Tumblr is (was?).

Am I best:

1. Having a single collection `Articles` mounted to the `Articles` page, with multiple blueprints (`link`, `image`, `video`, etc)
    - How can I pivot on the blueprint name in antlers? 
3. Having multiple collections (one per post type), all mounted to the `Articles` page
    - I assume I'd then do something like `{{ collection from="link|image|video" }}` to iterate through all together?
3. Have a `type` selector on the blueprint and use conditional field displaying based on the value, eg if type is `image` then hide the video field, etc

Thanks!

**Comment by jackmcdade** [comment, marked as answer]
I'd go with different blueprints — it's the same approach i took for the [Stumblr](https://statamic.com/starter-kits/statamic/stumblr) Starter Kit which is based on Tumblr. Give it a look!

**Reply by adamhopkinson** [reply]
Thanks @jackmcdade, this is great. I assume `{blueprint}` is a field rather than a variable?
